lib/rules/use-ember-get-and-set.js

```diff
@@ -121,7 +115,7 @@ module.exports = {
 
         // Skip this.get() and this.set() in tests/
         if (
-          isFileInDirectory('tests') &&
+          ember.isTestFile(filename) &&
```

Comment:
Not 100% sure if this is correct.. I removed the `isFileInDirectory` and saw this was using it, so I changed it to use what I thought was an equivalent 

I can revert it if it's not correct 😅

Reply:
This sounds good to improve consistency of test file checks.